internal/storage/db/check.go

```diff
@@ -0,0 +1,81 @@
+package db
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"github.com/kamencov/go-musthave-diploma-tpl/internal/customerrors"
+	"github.com/kamencov/go-musthave-diploma-tpl/internal/models"
+	"time"
+)
+
+func (d *DateBase) CheckTableUserLogin(ctx context.Context, login string) error {
```

Comment:
Предлагаю попробовать вот такой нейминг. Мне кажется сразу становится понятно, что метод делает?
```suggestion
func (d *DateBase) CheckUserExists(ctx context.Context, login string) error {
```